packages/cmrjs/cmr-utils.js

```diff
@@ -504,6 +504,23 @@ function mergeURLs(original, updated = [], removed = []) {
   return [...unchangedOriginals, ...updatedWithMergedOriginals];
 }
 
+/**
+ * Updates CMR JSON file with stringified 'metadataObject'
+ *
+ * @param {Object} metadataObject - JSON Object to stringify
+ * @param {Object} cmrFile - cmr file object to write body to
+ */
+async function uploadUMMGJSONCMRFile(metadataObject, cmrFile) {
+  const tags = await aws.s3GetObjectTagging(cmrFile.bucket, getS3KeyOfFile(cmrFile));
+  const tagsQueryString = aws.s3TagSetToQueryString(tags.TagSet);
+  return aws.promiseS3Upload({
+    Bucket: cmrFile.bucket,
+    Key: getS3KeyOfFile(cmrFile),
+    Body: JSON.stringify(metadataObject),
```

Comment:
Might be a good idea to add `ContentType: application/json` here.

Reply:
d4b13cb0d155f44de9dc0f6bb3b2d8f79d0cdee7